tests/runtests.pl

```diff
@@ -245,6 +245,7 @@ BEGIN
 my $has_multissl;   # set if curl is build with MultiSSL support
 my $has_manual;     # set if curl is built with built-in manual
 my $has_win32;      # set if curl is built for Windows
+my $has_mingw;      # set if curl is built with MinGW (as opposed to MinGW-w64)
```

Comment:
maybe we can use the chance to get a feature for classic MinGW and one for mingw-w64?

Reply:
I had that locally, but decided against including it if I don't need it...

Reply:
ok, maybe you can do a 2nd PR for this later on? if you had it working already, I think it would be good to have it for future tests.

Reply:
Isn't it too easy as an add-on when needed? I mean, it really is just copy-editing all the `has_mingw` parts, replacing the `-pc-mingw32` needle with a `-w64-mingw32` one (yes, also for 64-bit the suffix is `-w64-mingw32`).

When I read your suggestion, I also considered reversing the logic so that `mingw-w64` would be required, rather than `!MinGW`. But then I realized that the test runs just fine in the Visual Studio builds, so that would be too restrictive.